triton-vm/Cargo.toml

```diff
@@ -26,7 +26,7 @@ version = "1"
 default-features = false
 
 [dependencies]
-twenty-first = { version = "0.2.0" } # , path = "../twenty-first" }
+twenty-first = { git = "https://github.com/Neptune-Crypto/twenty-first.git", branch = "random-elements" }
```

Comment:
Before merging the present PR, merge this branch on twenty-first and change the dependency to the master branch.

Reply:
Why not make a new release on crates.io and depend on that? My impression was that depending on git repositories was unusual.

Reply:
I'd like to do that once the digest type has been added to twenty-first.